.github/workflows/tests.yml

```diff
@@ -63,3 +63,15 @@ jobs:
         run: docker run -d -p 7700:7700 getmeili/meilisearch:latest meilisearch --master-key=masterKey --no-analytics
       - name: Run integration tests
         run: go test -v ./...
+
+  code-cov:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with: { go-version: "1.17" }
+      - run: |
+          go test -race -coverprofile=coverage.txt -covermode=atomic ./...
+      - uses: codecov/codecov-action@v3
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I think you don't need a new job. Just add the coverage into the current `go test` call ;)

Reply:
I'll add the token in the settings!

Reply:
Done